src/App.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+body {
+  margin: 0;
+}
```

Comment:
- [ ] [OPTIONAL] I see you imported the Lato font. Please to make it work add `font-family: 'Lato', sans-serif;` to the body tag in CSS

Reply:
Thank you working on it now